docs/chaos-engineering/chaos-faults/aws/static/manifests/ec2-network-latency/destination-host-and-ip.yaml

```diff
@@ -0,0 +1,26 @@
+## it inject the chaos for the ingrees and egress traffic for specific ips/hosts
```

Comment:
```suggestion
# it injects the chaos into the egress traffic for specific IPs/hosts
```
We have removed mentions of `ingress` from the network experiments as `tc` only affects the `egress` to avoid user confusion.
Also single `#` to maintain consistency.